packages/webpack-cli/lib/groups/runHelp.js

```diff
@@ -73,6 +73,20 @@ const outputHelp = (args) => {
         process.exit(2);
     }
 
+    const updatedFlagEntries = flags.map((e) => {
```

Comment:
No `e`/`i`/`z`/any single character not in loop, never need to cut variable/method names, developer read it, not machines

Reply:
I will add eslint rule for it

Reply:
https://github.com/webpack/webpack-cli/blob/ed474c60af8d6e26b0647d9fa89dadd4ee9913be/packages/webpack-cli/lib/groups/runHelp.js#L167-L179